pkg/cmd/cli/cmd/project.go

```diff
@@ -19,7 +27,7 @@ func NewCmdProject(f *clientcmd.Factory, out io.Writer) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "project <project-name>",
 		Short: "switch to another project",
-		Long:  `Switch to another project and make it the default in your configuration.`,
+		Long:  `Switch to another project or context and make it the default in your configuration.`,
```

Comment:
Just say "project".  We'll define what contexts are when the user hits them.  Also, this should be a longer long description.